cmd/prysmctl/validator/proposerSettings.go

```diff
@@ -0,0 +1,100 @@
+package validator
```

Comment:
we tend to name our go files snake case `proposer_settings` vs camel case `proposerSettings`.